web-console/3-project-details/binding-in-project.md

```diff
@@ -51,39 +49,47 @@
-	- The success message for the "Create a secret in my project" option should say "Binding created successfully"
-- There is a note indicating that the binding operation created a configuration.
+- The success message for the "Create a secret in my project" option should say "Binding created successfully"
+- There is a note indicating that binding operation created a secret with a link to the secret.
 - Include an action to "View configuration" which would take the user to the page for that secret.
 - There should be a note alerting users that changes will not take effect until they redeploy their pods.
-- The only action button is "Close."
+- The only action is "Close."
 
 #### Implementation Details
 - Following the "View configuration" link would dismiss the overlay panel.
 
-
-## Unbinding
-- Unbinding would work similarly to binding, following the same two-step process:
+### Parameters Step
+![template](img/bind-parameters.png)
+- When binding from a service that takes bind parameters to an application, the wizard will display three steps: Binding, Parameters, Results.
+- When binding from an application, if the service selected in the Binding step takes parameters, an additional "Parameters" step is dynamically added to the wizard.
+- Parameters are shown similarly to the Configuration step of the [Ordering Experience](http://openshift.github.io/openshift-origin-design/web-console/5-components/order-from-catalog)
+- The primary action action for step 1 is "Next" and the primary action for step two is "Bind".
+- If the parameters are required, the "Bind" button is disabled until the required fields have been filled.
+- A secondary action "Back" is added but is only active on step 2.
+
+## Delete Binding
+- Delete Binding would work similarly to binding, following the same two-step process:
 	1. Choose the thing to unbind from
 	2. Confirmation
 
-![template](img/unbind_1.png)
+![template](img/delete-binding-select.png)
 - Available resources should be listed with radio buttons. The first resource should be selected by a default.
 - The primary action is "Unbind" and "Cancel" should be available as a secondary action.
 
-![template](img/unbind_2.png)
+![template](img/delete-binding-confirm.png)
 - The success message should name both the application and the provisioned service that were bound.
 - There should be a note alerting users that changes will not take effect until they redeploy their pods.
 - The only action button is "Close."
 
-## Deprovisioning
-- The "Deprovision" action is available in the kebab regardless whether or not a service has any bindings.
-- Clicking the "Deprovision" action brings up a strong warning.
+## Delete Service
+- The "Delete" action is available in the kebab regardless whether or not a service has any bindings.
+- Clicking the "Delete" action brings up a dialog with a strong warning.
 
-![template](img/deprovision_warning.png)
+![template](img/delete-warning.png)
```

Comment:
@ncameronbritt I don't think we want inline notifications inside of a popup, do we?